runtime/test/manifest-test.js

```diff
@@ -1316,11 +1320,63 @@ resource SomeName
     `);
 
     let [recipe] = manifest.recipes;
+    assert(recipe.normalize());
+    assert(recipe.isResolved());
+    const schema = recipe.particles[0].connections.bar.type.entitySchema;
+    const innerSchema = schema.fields.foo.schema.model.entitySchema;
+    assert.deepEqual(innerSchema.fields, {far: 'Text'});
+
     assert.equal(manifest.particles[0].toString(),
 `particle P in 'null'
   in Bar {Reference<Foo {Text far}> foo} bar
   affordance dom`);
   });
+  it('can resolve a particle with a collection of schema references', async () => {
+    let manifest = await Manifest.parse(`
+      schema Foo
+        Text far
+      particle P
+        in Bar {[Reference<Foo>] foo} bar
+      recipe
+        create as handle
+        P
+          bar = handle
+    `);
+
+    let [recipe] = manifest.recipes;
+    assert(recipe.normalize());
+    assert(recipe.isResolved());
+    const schema = recipe.particles[0].connections.bar.type.entitySchema;
+    const innerSchema = schema.fields.foo.schema.schema.model.entitySchema;
```

Comment:
Are you happy with schema.schema? It's not clear to me what they refer to exactly, but I'm leaving this to you.

Reply:
Not that happy but changing it is a big job.. I'll put it in the list of technical debt.